tvx/exec_lotus.go

```diff
@@ -0,0 +1,78 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/davecgh/go-spew/spew"
+	"github.com/filecoin-project/lotus/chain/types"
+	"github.com/filecoin-project/lotus/lib/blockstore"
+	"github.com/ipld/go-car"
+	"github.com/urfave/cli/v2"
+
+	"github.com/filecoin-project/oni/tvx/lotus"
+)
+
+var execLotusCmd = &cli.Command{
+	Name:        "exec-lotus",
+	Description: "execute a test vector against Lotus",
+	Flags:       []cli.Flag{&fileFlag},
+	Action:      runExecLotus,
+}
+
+func runExecLotus(c *cli.Context) error {
+	f := c.String("file")
```

Comment:
prefer `
```suggestion
	f := c.String(fileFlag.Name)
```

Reply:
In a recent commit, I changed the way we consume flags. We're now using the `Destination` feature, so no need to consume flags from the context explicitly (at least for now, it may change when we do more complex stuff going forward).